server/chat-commands/moderation.ts

```diff
@@ -189,7 +189,9 @@ export const commands: ChatCommands = {
 			try {
 				shouldPopup = runPromote(user, room, userid, nextSymbol, toPromote, force);
 			} catch (err) {
-				if (err.name?.endsWith('ErrorMessage')) this.errorReply(err.message);
+				// Throw actual crashes, but don't throw chat ErrorMessages
+				if (!err.name?.endsWith('ErrorMessage')) throw err;
+				this.errorReply(err.message);
 				continue;
```

Comment:
```suggestion
				if (err.name?.endsWith('ErrorMessage')) {
					this.errorReply(err.message);
					continue;
				}
				throw err;
```

This is how I'd implement this.